backends/aerospike.go

```diff
@@ -110,6 +85,45 @@ func NewAerospikeBackend(cfg config.Aerospike, metrics *metrics.Metrics) *Aerosp
 	}
 }
 
+// generateAerospikeClientPolicy returns an Aerospike ClientPolicy object configured according to values
+// in config.Aerospike fields
+func generateAerospikeClientPolicy(cfg config.Aerospike) *as.ClientPolicy {
+	clientPolicy := as.NewClientPolicy()
+	// cfg.User and cfg.Password are optional parameters
+	// if left blank in the config, they will default to the empty
+	// string and be ignored
+	clientPolicy.User = cfg.User
+	clientPolicy.Password = cfg.Password
+
+	// Aerospike's connection idle deadline default is 55 seconds. If greater than zero, this
+	// value will override
```

Comment:
I think you can remove the `if greater than zero, this value will override` from this comment